pkg/core/native/native_nep17.go

```diff
@@ -45,9 +45,12 @@ func (c *nep17TokenNative) Metadata() *interop.ContractMD {
 	return &c.ContractMD
 }
 
-func newNEP17Native(name string, id int32) *nep17TokenNative {
-	n := &nep17TokenNative{ContractMD: *interop.NewContractMD(name, id, func(m *manifest.Manifest) {
+func newNEP17Native(name string, id int32, onManifestConstruction ...func(m *manifest.Manifest, hf config.Hardfork)) *nep17TokenNative {
```

Comment:
Variadic? Well, can be, of course...

Reply:
Just in order not to modify other usages. But I'm OK with both ways, fixed.